systest/backup/nfs-backup/backup_test.go

```diff
@@ -49,6 +50,13 @@ var (
 )
 
 func TestBackupHAClust(t *testing.T) {
+	// check health of alpha1
+	for i := 1; i <= 50; i++ {
+		if err := testutil.CheckHealthContainer(testutil.ContainerAddr("alpha1", 8080)); err == nil {
+			break
+		}
+		time.Sleep(2 * time.Second)
```

Comment:
nitpick - change from `2*time.second` to `1*time.second` && increase the for loop from `i<=50` to `I<=100`. 

this would be more readable.

Reply:
done